backend/mobilityAI/Controllers/MobilityAIController.cs

```diff
@@ -129,18 +138,19 @@ public JsonResult GetRangeGyroscope(long start, long end)
         /// <param name="GyroscopeFile">
         /// The file input for Gyroscope
         /// </param>
-        /// <param name="DeviceId">
-        /// The device id from which this data came
+        /// <param name="PatientID">
+        /// The patient id from which this data came from
         /// </param>
         [HttpPost("AddData", Name = "AddData")]
-        public async Task<IActionResult> AddData(IFormFile AccelerometerFile, IFormFile GyroscopeFile, string DeviceId)
+        public async Task<IActionResult> AddData(IFormFile AccelerometerFile, IFormFile GyroscopeFile, int PatientId)
```

Comment:
Keep the style for the argument names consistent. Eg: here it's `PatientId` but [there](https://github.com/pgunasekara/4zp6/pull/84/files#diff-d132b998e7be1c2676766c2c2e31cd6cR123) it's `patientId`